[WIP] Theme tweaks

amycatgirl opened the pull request:
~~Now actually stacked on top of #10~~ merged

Commit:
fix: icons having low contrast on dark mode

Commit:
chore: don't ignore envrc (direnv)

Commit:
chore: nix develop

sorry i just had to do it
i swear i don't have an addiction
i swearrr

retcinder commented:
Shouldn't this be targeting main as a base instead of the experiment whatiskillingfrontend branch?

Rexogamer commented:
yes (and that branch can probably be deleted now). fixed

lo2dev committed:
Tweak category btn background

lo2dev committed:
Make icons follow theme

Also remove icon backgrounds for settings rows
Also remove the background for the profile badges in settings

amycatgirl commented:
Conflicts with recent changes in the latest rebase, superseded by #14
